user_modules/IsReady_Template/testCasesForTTS_IsReady.lua

```diff
@@ -0,0 +1,271 @@
+          text_field("navigationText"),
+          text_field("audioPassThruDisplayText1"),
+          text_field("audioPassThruDisplayText2"),
+          text_field("sliderHeader"),
+          text_field("sliderFooter"),
+          text_field("notificationText"),
+          text_field("menuName"),
+          text_field("secondaryText"),
+          text_field("tertiaryText"),
+          text_field("timeToDestination"),
+          text_field("turnText"),
+          text_field("menuTitle"),
+          text_field("locationName"),
+          text_field("locationDescription"),
+          text_field("addressLines"),
+          text_field("phoneNumber")
+        },
+        imageFields =
+        {
+          image_field("softButtonImage"),
+          image_field("choiceImage"),
+          image_field("choiceSecondaryImage"),
+          image_field("vrHelpItem"),
+          image_field("turnIcon"),
+          image_field("menuIcon"),
+          image_field("cmdIcon"),
+          image_field("showConstantTBTIcon"),
+          image_field("locationImage")
+        },
+        mediaClockFormats =
+        {
+          "CLOCK1",
+          "CLOCK2",
+          "CLOCK3",
+          "CLOCKTEXT1",
+          "CLOCKTEXT2",
+          "CLOCKTEXT3",
+          "CLOCKTEXT4"
+        },
+        graphicSupported = true,
+        imageCapabilities = { "DYNAMIC", "STATIC" },
+        templatesAvailable = { "TEMPLATE" },
+        screenParams =
+        {
+          resolution = { resolutionWidth = 800, resolutionHeight = 480 },
+          touchEventAvailable =
+          {
+            pressAvailable = true,
+            multiTouchAvailable = true,
+            doublePressAvailable = false
+          }
+        },
+        numCustomPresetsAvailable = 10
+      },
+      audioPassThruCapabilities =
+      {
+        samplingRate = "44KHZ",
+        bitsPerSample = "8_BIT",
+        audioType = "PCM"
+      },
+      hmiZoneCapabilities = "FRONT",
+      softButtonCapabilities =
+      {
+        {
+          shortPressAvailable = true,
+          longPressAvailable = true,
+          upDownAvailable = true,
+          imageSupported = true
+        }
+      }
+    })
+  :Times(exp_occur)
+  :Timeout(20000)
+
+  ExpectRequest("VR.IsReady", true, { available = true })
+  -- ExpectRequest("TTS.IsReady", true, { available = true })
+  ExpectRequest("UI.IsReady", true, { available = true })
+  ExpectRequest("Navigation.IsReady", true, { available = true })
+  ExpectRequest("VehicleInfo.IsReady", true, { available = true })
+
+  self.applications = { }
+  ExpectRequest("BasicCommunication.UpdateAppList", false, { })
+  :Pin()
+  :Do(function(_, data)
+      self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", { })
+      self.applications = { }
+      for _, app in pairs(data.params.applications) do
+        self.applications[app.appName] = app.appID
+      end
+    end)
+
+  self.hmiConnection:SendNotification("BasicCommunication.OnReady")
+end
```

Comment:
@vvvakulenko, because of ATF issue, when Times(0) specified timeout is not waited from ATF. To go aorund this problem, please add before end waiting of 20 sec:

commonTestCases:DelayedExp(20000)
Please don't forget to require in start of script commonTestCases